src/mmw/js/src/analyze/views.js

```diff
@@ -226,7 +226,7 @@ var TableRowView = Marionette.ItemView.extend({
         return {
             // Convert coverage to percentage for display.
             coveragePct: (this.model.get('coverage') * 100).toFixed(1),
-            areaTrimmed: this.model.get('area').toFixed(1)
+            area: this.model.get('area')
```

Comment:
I don't think we need this anymore since the model should be passed to the view automatically.